packages/core/status/core-status-server-internal/src/routes/status.ts

```diff
@@ -48,6 +48,10 @@ interface StatusHttpBody extends Omit<StatusResponse, 'status'> {
   status: StatusInfo | LegacyStatusInfo;
 }
 
+interface RedactedStatusHttpBody {
+  status: string;
+}
```

Comment:
This diverges from the response format of `/api/status` when authenticated. Is that what we want? When someone is unauthenticated with `allowAnonymous: false`, they'll get:

```js
{
  "status": "available"
}
```

When authenticated, they'll get

```js
{
  ...,
  "status": {
    ...,
    "overall": {
      ...,
      "level": "available",
    }
  }
}
```

Would we want the structure to be consistent, to minimize confusion if someone is integrating with this API without auth and chooses to add auth later, or vice-versa?

Reply:
Same question for Preboot as well - do we want this to diverge from the normal/non-preboot `/api/status`?

Reply:
> This diverges from the response format of /api/status when authenticated. Is that what we want?

Well, this is what we all agreed on in the related issue 😄. But I agree that keeping the same "structure" is probably better (even if that wouldn't really solve the problem of any integration trying to access any other property in the response's payload...)

Reply:
Also note that the `ServiceStatus` type returned with the full version has fields that we want to redact for the anon version (basically ALL fields except from `level`), for security reasons 

https://github.com/elastic/kibana/blob/12466d8b17d8557ff0b561c346511bd1760da4c1/packages/core/status/core-status-common/src/service_status.ts#L18-L40

So instead of the PR's current:

```ts
XXX { status: "status" }
```

We would have:

```ts
XXX { status: { overall: { level: "status" }}}
```

Are we ok with that?

Reply:
done in https://github.com/elastic/kibana/pull/159768/commits/b4fe7932486b1178dd976ac02d8289cc2fea4f5c

Reply:
```{ status: { overall: { level: "status" }}}``` is more verbose than what's necessary, but I think I agree with Luke here that even if redacted, having the same shape seems preferable so that anon is bwc with authenticated (even if we can't say the opposite)